xbmc/cores/VideoPlayer/VideoPlayerRadioRDS.h

```diff
@@ -15,6 +15,7 @@
 #include "utils/Stopwatch.h"
 
 #include <deque>
+#include <memory>
 
 class CDVDStreamInfo;
```

Comment:
Forward decl for `CPVRRadioRDSInfoTag` and `CPVRChannel` missing; works only by luck, becaase FileItem.h gets included (and because it carries unneeded forward declarations!).